tests/integration/test_collect.py

```diff
@@ -1,20 +1,16 @@
-import shutil
 import csv
 import urllib.request
 import json
-import hashlib
 from datetime import datetime
-from helpers import execute
-from pathlib import Path
+from tests.utils.helpers import execute, hash_digest
 
 import pytest
 
-ENDPOINT = "https://www.registers.service.gov.uk/registers/country/download-csv"
-COLLECTION_DIR = "./collection"
+ENDPOINT = "https://raw.githubusercontent.com/digital-land/digital-land-python/main/tests/data/resource_examples/csv.csv"
 
 
 @pytest.fixture()
-def endpoint_csv(tmp_path):
+def create_endpoint_csv(tmp_path):
```

Comment:
again avoid imperative language in fixture names. "test_collect is passed an endpoint_csv"